Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -187,8 +187,10 @@ private void OnMapInit(EntityUid uid, SuitSensorComponent component, MapInitEven
 
     private void OnEquipped(EntityUid uid, SuitSensorComponent component, ref ClothingGotEquippedEvent args)
     {
-        if (HasComp<DisableSuitSensorsComponent>(args.Wearer)) // Frontier: entities with disabled suit sensors must never be set as a valid user.
-            return; // Frontier
+        // Frontier: entities with "Always" disabled suit sensors must never be set as a valid user.
+        if (TryComp<DisableSuitSensorComponent>(args.Wearer, out var disableSuitSensor) && disableSuitSensor.Always)
+            return;
```

Comment:
```suggestion
        if (TryComp<DisableSuitsSensorComponent>(args.Wearer, out var disableSuitSensor) && disableSuitSensor.RemoveRegistration)
            return;
```
rename consistently vs. component